js/foam/ui/md/Halo.js

```diff
@@ -215,7 +229,27 @@ CLASS({
         if (this.focusType === 'mouse') return;
 
         this.focusType = 'keyboard';
+        // Save all of the halo properties.
+        const save = {
+          animateGrowth: this.animateGrowth,
+          startAlpha: this.startAlpha,
+          easeInTime: this.easeInTime,
+          easeOutTime: this.easeOutTime
```

Comment:
Add a trailing comma here.

Reply:
https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma

Reply:
Done, thank you!